src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -3109,47 +3109,60 @@ namespace winrt::TerminalApp::implementation
         // Begin Theme handling
         _updateThemeColors();
 
+        const auto tab{ _GetFocusedTab() };
+        _updateTabsCloseButton(tab);
```

Comment:
nit: no extra var necessary
```suggestion
        _updateTabsCloseButton(_GetFocusedTab());
```